tests/modules/nf-core/mapad/map/main.nf

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { MAPAD_INDEX } from '../../../../../modules/nf-core/mapad/index/main.nf'
+include { MAPAD_MAP } from '../../../../../modules/nf-core/mapad/map/main.nf'
+
+//
+// Test with single-end data
```

Comment:
And the test for paired-end? Or is that not supported?

Reply:
Might be worth specifying it if so (reads must be single-end and merged, i.e. non-merged paired-end input is not allowed)